docs/data-protection/data-protection-statement-website.md

```diff
@@ -9,7 +9,9 @@
 > It's painful - but you will have to go through the entire text and adjust it to your own company website
 > where necessary. We hope this template still hopes you with a general structure.
 
-# Website Data Protection Statement for Data Subjects Pursuant to Articles 13 and 14 of the EU General Data Protection Regulation
+# Website Data Protection Statement
+
+For Data Subjects Pursuant to Articles 13 and 14 of the EU General Data Protection Regulation
 
 Data protection is a particularly high priority for the management of \<enter company name\>. The use of the
```

Comment:
I thought you had a macro for this (company name)

Reply:
Yup, I do. The was just me copying templates across though. There's heaps of items that I'll need to replace with macros. That will be something I'll be addressing as I go through each document in:

https://github.com/RadiotherapyAI/RadiotherapyAI/issues/62